types/katex/contrib/katex2tex.d.ts

```diff
@@ -0,0 +1,23 @@
+declare module 'katex/contrib/copy-tex/katex2tex' {
+    /**
+     * Set these to how you want inline and display math to be delimited.
+     */
+    interface ICopyDelimiters {
```

Comment:
Recommend export param interface and named without 'I'.
Such as `export interface CopyDelimiters {`.